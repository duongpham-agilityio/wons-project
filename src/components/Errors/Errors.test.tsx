import { render } from '@testing-library/react';
import Error from '.';

describe('Error component', () => {
  test('renders error', () => {
    const { container } = render(<Error />);
    expect(container).toMatchSnapshot();
  });
});
