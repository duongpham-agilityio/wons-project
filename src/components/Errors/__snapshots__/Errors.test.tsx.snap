// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component renders error 1`] = `
<div>
  <div
    class="container max-w-lg bg-white p-8 flex flex-col items-center justify-center h-full m-auto"
  >
    <h1
      class="text-3xl font-bold text-red-500 mb-4"
    >
      Oops! Something went wrong.
    </h1>
    <p
      class="text-gray-600 text-xl"
    />
    <a
      aria-label="Go to Home Page"
      class="pt-5 text-blue-400 text-xl hover:underline"
      href="/"
    >
      Go to Home Page
    </a>
  </div>
</div>
`;
